gateway-getting-started-guide-turris-omnia.md

```diff
@@ -0,0 +1,79 @@
+# Getting Started
+
+## WebThings Gateway for Turris Omnia
+
+**Warning:** WebThings Gateway for Turris Omnia builds are currently highly experimental. So far the WebThings Gateway web interface only provides very basic router settings and it is not suitable for replacing your existing router software. We do not recommend using these builds for your primary home router.
+
+<img src="/images/turris_omnia.png" width="400">
+
+[WebThings Gateway](https://iot.mozilla.org/gateway/) is a software distribution for smart home gateways which allows users to directy monitor and control their smart home over the web, without a middleman.
+
+### What you will need
+
+1. **A [Turris Omnia](https://www.turris.cz/en/omnia/)** router
+2. **A USB stick** (at least 256MB)
```

Comment:
This doesn't say how the stick should be formatted (although I'm guessing it's FAT and for most sticks that will be the default anyway)

Reply:
To be honest, I don't actually know which is why I haven't specified! But yes, FAT seems two work.

Reply:
We should ask @dhylands and update later.